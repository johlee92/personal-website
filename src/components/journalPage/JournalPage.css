.journalPage {
    width: 100%;
    display: flex;
    flex-direction: column;
    align-items: center;
    white-space: pre-line;
}

.journalPage-container {
    margin: 10px 0;
    width: 80%;
    max-width: 720px;
}