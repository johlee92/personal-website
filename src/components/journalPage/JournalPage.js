import React, { Component } from 'react';
import './JournalPage.css';

class JournalPage extends Component {
    render() {
        return (
            <div>
            </div>
        )
    }
}

export default JournalPage;