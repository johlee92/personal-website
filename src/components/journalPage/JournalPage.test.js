import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import JournalPage from './JournalPage';

describe(`JournalPage component`, () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
                <JournalPage />
            , div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the component as expected', () => {
        const tree = renderer
            .create(
                <JournalPage />
            )
            .toJSON();
        expect(tree).toMatchSnapshot();  
    });
})
