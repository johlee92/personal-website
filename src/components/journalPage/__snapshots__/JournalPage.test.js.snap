// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JournalPage component renders the component as expected 1`] = `<div />`;
